integrations/gitlab/gitlab_integration/gitlab_service.py

```diff
@@ -404,18 +405,34 @@ async def get_all_root_groups(self) -> typing.AsyncIterator[List[Group]]:
         def is_root_group(group: Group) -> bool:
             return group.parent_id is None
 
+        cached_groups = event.attributes.setdefault(GROUPS_CACHE_KEY, {}).setdefault(
```

Comment:
wouldn't you want to cache regular groups as well and not only root? see function above